lib/rules/no-try-catch.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @fileoverview Rule to flag use of with statement
+ * @author Nicholas C. Zakas
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+    return {
+        "TryStatement": function(node) {
+            context.report(node, "Unexpected use of 'try' statement.");
+            if (node.handler && node.handler.type === "CatchClause") {
```

Comment:
Given that you can't have a `catch` or `finally` without a `try`, why do you need to report each one separately?